### DISCUSSION TITLE
Application name conflict.

### DISCUSSION THREAD

**Discussion by setanarut** [sbarex/QLMarkdown#18] [Ideas]
This name exists on homebrew. (`$ brew install --cask qlmarkdown`) It can be replaced by a more catchy special name. for example mdlook, MDLook or  qlmarkdown2,  there is [qlmarkdowngfm](https://github.com/Watson1978/QLMarkdownGFM)

**Comment by sbarex** [comment]
Oh... I went crazy correcting the typo on the name and now I have to exchange it again ...

**Reply by setanarut** [reply]
i  will design a logo and icon after your final name

**Comment by setanarut** [comment]
@sbarex If the name will not change, I will make a logotype. Is QLMarkdown the app name you decided on?

**Comment by sbarex** [comment]
I haven't thought much about the name. I would leave the QL prefix. If you have any suggestions, they are welcome!

**Comment by setanarut** [comment]
qlmarkdown is fine. Installation with homebrew would be nice when the application is stable. `brew install --cask ql-markdown`

**Comment by setanarut** [comment]
related -> https://github.com/Homebrew/homebrew-cask/pull/98284
